Data Prepper 2.15 release blog (#4114)
opensearch-project/project-website: _community_members/srikanthpadakanti.md

---
short_name: 'srikanthpadakanti'
name: 'Srikanth Padakanti'
photo: '/assets/media/community/members/srikanthpadakanti.jpeg'
job_title_and_company: 'Senior Software Engineer at Apple'
primary_title: 'Srikanth Padakanti'
title: 'OpenSearch Community Member: Srikanth Padakanti'
breadcrumbs:
  icon: community
  items:
    - title: Community
      url: /community/index.html
    - title: Members
      url: /community/members/index.html
    - title: "Srikanth Padakanti&apos;s Profile"
      url: '/community/members/srikanth-padakanti.html'
keynote_speaker: false
linkedin: 'srikanth-padakanti-9023b720'
github: 'srikanthpadakanti'
permalink: '/community/members/srikanth-padakanti.html'
personas:
  - author
redirect_from: '/authors/srikanthpadakanti/'
---

> Srikanth is a senior software engineer at Apple, where he architects and builds large scale data platforms and infrastructure that power critical enterprise systems. His work at Apple focuses on designing high throughput data pipelines, search infrastructure, and observability solutions that operate at massive scale across distributed environments. He is an active contributor to the OpenSearch project across multiple repositories including OpenSearch core, SQL, Data Prepper, ML Commons, and others. His contributions span the full stack, from core search engine internals and query language capabilities to data ingestion pipelines and metrics integration, building reliable systems that operate at production scale. Passionate about open source and continuous learning, Srikanth is dedicated to advancing the OpenSearch ecosystem and staying at the forefront of distributed systems and observability. Srikanth holds a Master's degree in Computer Science from Texas A&M University.

Reviewer: ```suggestion
Srikanth is a senior software engineer at Apple, where he architects and builds large-scale data platforms and infrastructure that power critical enterprise systems. His work at Apple focuses on designing high-throughput data pipelines, search infrastructure, and observability solutions that operate at massive scale across distributed environments. He is an active contributor to OpenSearch across multiple repositories, such as OpenSearch core, SQL, Data Prepper, and ML Commons. His contributions span the full stack, from core search engine internals and query language capabilities to data ingestion pipelines and metrics integration. Passionate about open source and continuous learning, Srikanth is dedicated to advancing OpenSearch and staying at the forefront of distributed systems and observability. Srikanth holds a Master's degree in computer science from Texas A&M University.
```
